src/main/java/no/ndla/taxonomy/domain/Node.java

```diff
@@ -323,7 +421,7 @@ public Collection<Node> getChildNodes() {
                 .toList();
     }
 
-    public Collection<Node> getParentNodes() {
+    public List<Node> getParentNodes() {
```

Comment:
Noen spesiell grunn til at denne er endra?

Reply:
Var for å få koden i crudcontroller litt penere, men tror jeg kunne gjøre det på en annen måte :shrug: :smile: